### PR TITLE
[11.0][IMP] project_task_dependency: Auto arranging

### PR DESCRIPTION
**Pull request by tarteo**
Adds an setting which enables arranging start and end dates of tasks based on their dependencies. This feature is optional and can be turned on in the project settings.

auto arrange

### CODE REVIEW

File: project_task_dependency/__manifest__.py [hunk 1]
```diff
@@ -1,9 +1,9 @@
-# © 2016 Onestein (<http://www.onestein.eu>)
+# Copyright 2016 Onestein (<http://www.onestein.eu>)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
     'name': 'Project Task Dependencies',
-    'version': '11.0.1.0.1',
+    'version': '11.0.1.1.1',
```
Comment (astirpe) [inline project_task_dependency/__manifest__.py]: Should be `'11.0.1.1.0'`

File: project_task_dependency/models/__init__.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
-# © 2016 Onestein (<http://www.onestein.eu>)
+# Copyright 2016 Onestein (<http://www.onestein.eu>)
```
Comment (astirpe) [inline project_task_dependency/models/__init__.py]: No need of Copyright in the `__init__.py` files.

**Review by astirpe — CHANGES_REQUESTED** [review]

**Review by astirpe — APPROVED** [review]

**Comment by astirpe** [comment]
@tarteo it would be nice if you can add some screenshots that show the effect of this feature in a timeline view.
This module provides a feature that is useful but also very cool if you see it in action on the timeline. :wink:

**Comment by tarteo** [comment]
I've updated the description with a little video of how it works.

**Comment by pedrobaeza** [comment]
I'm not sure about this, as this is not taking into account weekends and days off. I think we should move this to the schedule planning module we are doing. What do you think?

**Comment by tarteo** [comment]
@pedrobaeza If the arranging can be done on the fly on tasks, then I agree. I'll keep my branch so you can use the code of it if you want.

**Comment by pedrobaeza** [comment]
Not yet, but it can be added using current module methods for skipping days off.

**Comment by pedrobaeza** [comment]
Are you going to continue with this? The branch has conflicts.

**Comment by tarteo** [comment]
@pedrobaeza I have no time for this at the moment, I'll close this one and try to put it project_schedule another time.

**Comment by tarteo** [comment]
And with the addition of https://github.com/OCA/web/pull/1024 this is not really adding much anymore.

**Comment by pedrobaeza** [comment]
Certainly...

**Pull request closed without merging**